openqabot/smeltsync.py

```diff
@@ -116,10 +116,7 @@ def _create_record(cls, inc):
         incident["approved"] = approved
         incident["rr_number"] = rr_number
         incident["inReviewQAM"] = inReviewQAM
-        # this is temporary solution. At first stage we will just push this field into dashboard
-        # next we will add logic to store it in dashboard and then finally we will replace
-        # here dummy value with real one from smelt
-        incident["embargoed"] = False
+        incident["embargoed"] = inc["crd"][0]["crd"]
```

Comment:
what type is `inc["crd"][0]["crd"]` and what happens if is "crd" missing ?

Reply:
1. crd suppose to be boolean 
2. Regarding what will happen if field will be missing I think same question can be applied to ALL fields in `inc` , right ? Don't see why `crd` field should have special treatment here . All fields which we got here if I understand everything right are defined by `INCIDENT` query in smelt.py . So removal of any field from this query will cause exceptions in this function ( and in many other places ) .

Reply:
1 ... `crd` according docs is `DateTime` and api of dashboard excepts boolean ...

Reply:
1) `crd`  according to docs is `DateTime`  ...

Reply:
during initial discussions I remember statement that we can not rely on this date , because actual agreement saying " until customer get the update first " which normally happening before date defined in crd but not always and in THAT case we should obey actual delivery not this date . 

So agreement which I remember was that **everything** what has this date should be blocked no matter if date there in the past or future . Just the fact of something defined in this field give us true